.github/workflows/insiders.yml

```diff
@@ -31,7 +31,7 @@ env:
 jobs:
   build-vsix:
     name: Build VSIX
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```

Comment:
Build on Windows, since the napi libraries are only needed on Windows. So we don't need a build step to generate the binaries for all platforms.

Reply:
Does this mean we have to ship the windows binaries? So this HAS to build on windows? If so, we need this in the release.yml too.